dosed/datasets/dataset.py

```diff
@@ -410,22 +407,62 @@ def __init__(self,
         )
         self.ratio_positive = ratio_positive
 
+        self.index_to_record_event = []
```

Comment:
C'est pas bon ça j'ai l'impression que c'est l'ancienne version du get sample n on ?

Reply:
I changed the place of this function but it's the good version of it